src/Components/SmartComponents/Reports/Common/tablePage.js

```diff
@@ -74,7 +76,7 @@ const tablePage = ({ data, page, intl, header }) => {
             hitColumns.push('status');
         }
     }
-    else {
+    else if (type === 'cves') {
```

Comment:
This component should be more abstract and less "smart". For example, you "can't have" multiple if statements for each table header you will render.  The component should be responsible only for rendering the table